provisioning/roles/discourse/templates/app.yml.j2

```diff
@@ -51,16 +51,18 @@ env:
 
   ## TODO: List of comma delimited emails that will be made admin and developer
   ## on initial signup example 'user1@example.com,user2@example.com'
-  DISCOURSE_DEVELOPER_EMAILS: 'matthew@oaf.org.au,james@oaf.org.au'
+  DISCOURSE_DEVELOPER_EMAILS: 'matthew@oaf.org.au,james@oaf.org.au,ian@oaf.org.au,simon@oaf.org.au,brenda@oaf.org.au'
```

Comment:
Maybe this list should come from one place,......  Like a list of staff.

But this is good enough for now.